app/main/views/frameworks.py

```diff
@@ -84,6 +85,11 @@ def framework_dashboard(framework_slug):
     framework_dates = content_loader.get_message(framework_slug, 'dates')
     framework_urls = content_loader.get_message(framework_slug, 'urls')
 
+    try:
+        use_of_data = content_loader.get_message(framework_slug, 'use-of-data')
```

Comment:
Probably super-picky, but the 'block' (according to `content_loader.ContentLoader#get_message`) is supposed to be the "place" where the message is used. Whereas the 'key' (if present) is the specific message.

So we have our naming wrong here, maybe. Probably doesn't matter.